pkg/tfbridge/provider.go

```diff
@@ -1684,7 +1684,7 @@ func transformFromState(
 func newTimeoutOverrides(key shim.TimeoutKey, maybeTimeoutSeconds float64) map[shim.TimeoutKey]time.Duration {
 	timeoutOverrides := map[shim.TimeoutKey]time.Duration{}
 	if maybeTimeoutSeconds != 0 {
-		timeoutOverrides[shim.TimeoutCreate] = time.Duration(maybeTimeoutSeconds * float64(time.Second))
+		timeoutOverrides[key] = time.Duration(maybeTimeoutSeconds * float64(time.Second))
```

Comment:
Shouldn't we have a lint for unused function args? That would have caught this.

Reply:
😢 
https://github.com/pulumi/pulumi-terraform-bridge/blob/22fdb29205f4415e371a3dc12165334fe59fc50d/.golangci.yml#L26

Reply:
I opened https://github.com/pulumi/pulumi-terraform-bridge/issues/1750.

Reply:
Nice.